### PR TITLE
Control Plane Logs Collection for OCNE and Standalone Kubernetes Clusters

### PR DESCRIPTION
**Pull request by naga-barri**

### CODE REVIEW

**Review by santhoshkvuda — DISMISSED** [review]

**Review by palukuri — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 2023-11-07
+### Added
+- Control Plane Logs Collection for OCNE and Standalone Kubernetes Clusters.
+- Support for launching Fluentd containers in privileged mode (default false).
```
Comment (kgvarun) [inline CHANGELOG.md:6]: Is this for both DS and RS?

File: charts/logan/values.yaml [hunk 2]
```diff
@@ -67,6 +67,12 @@ kubernetesClusterID:
 # e.g. production-cluster
 kubernetesClusterName:
 
+# -- Kubernetes Security Context privileged flag
+# By default, this is false and not required to be set in OKE cluster
+# In some special Kubernetes environments where SELinux mode is enforcing, we require this flag to be set to ture
```
Comment (kgvarun) [inline charts/logan/values.yaml:72]: Suggest rephrasing:
"Default: 'false'. This is not a required for OKE clusters. In Kubernetes environments where SELinux mode is enforced, set this flag to 'true' to allow fluentd pods to access log files.”

File: docs/FAQ.md [hunk 3]
```diff
@@ -275,3 +275,60 @@ oci-onm-logan:
       worker: 1
 ```       
 
+### Log Collection for OCNE (Oracle Cloud Native Environment)
+
+#### How to fix _execution expired_ error ?
```
Comment (kgvarun) [inline docs/FAQ.md:280]: Pls add where might users see this error message.

File: docs/FAQ.md [hunk 4]
```diff
@@ -275,3 +275,60 @@ oci-onm-logan:
       worker: 1
 ```       
 
+### Log Collection for OCNE (Oracle Cloud Native Environment)
```
Comment (kgvarun) [inline docs/FAQ.md:278]: On all troubleshoot related faqs pls add the location of files/etc where users might see these error messages.

**Review by kgvarun — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (naga-barri) [reply CHANGELOG.md:6]: Yes

**Review by kgvarun — APPROVED** [review]

**Review by palukuri — APPROVED** [review]

**Review by santhoshkvuda — APPROVED** [review]

**Pull request merged**